haystack/document_stores/memory.py

```diff
@@ -20,7 +27,8 @@
 logger = logging.getLogger(__name__)
 
 
-class InMemoryDocumentStore(BaseDocumentStore):
+class InMemoryDocumentStore(KeywordDocumentStore):
+    # pylint: disable=R0904
```

Comment:
pylint was complaining about too many public methods (23 instead of 20), so I disabled this particular check for this class. Any better idea?

Reply:
No, that's alright! :+1: